content/en/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md

```diff
@@ -41,7 +41,7 @@ These repo labels let reviewers filter for PRs and issues by language. For examp
 
 ### Team review 
 
-L10n teams can now review and approve their own PRs. For example, review and approval permissions for English are [assigned in an OWNERS file](https://github.com/kubernetes/website/blob/master/content/en/OWNERS) in the top subfolder for English content. 
+L10n teams can now review and approve their own PRs. For example, review and approval permissions for English are [assigned in an OWNERS file](https://github.com/kubernetes/website/blob/main/content/en/OWNERS) in the top subfolder for English content. 
```

Comment:
Generally we don't update blog posts that are older than a year-we leave them as an archive. In this case, as the master link redirects to the main link automatically (due to GitHub's redirect system), I don't think we should make this change here.